integration_tests/test_builtin_type.py

```diff
@@ -6,6 +6,8 @@ def test_builtin_type():
     s: str = "Hello, LPython!"
     l: list[i32] = [1, 2, 3, 4, 5]
     d: dict[str, i32] = {"a": 1, "b": 2, "c": 3}
+    t: tuple[str, i32] = ("a", 1)
+    st: set[i32] = {1, 2, 3, 4}
```

Comment:
You will need to remove this `set` test from here.

Reply:
I am really sorry for the above mistakes @Shaikh-Ubaid. I was in a hurry and pushed unfinished changes. I am fixing this.